src/components/tx/modals/TokenTransferModal/ReviewMultisigTx.tsx

```diff
@@ -0,0 +1,46 @@
+import { type ReactElement } from 'react'
+import { Box, Typography } from '@mui/material'
+import type { SafeTransaction } from '@gnosis.pm/safe-core-sdk-types'
+
+import SignOrExecuteForm from '@/components/tx/SignOrExecuteForm'
+import { createTokenTransferParams } from '@/services/tx/tokenTransferParams'
+import useBalances from '@/hooks/useBalances'
+import useAsync from '@/hooks/useAsync'
+import { createTx } from '@/services/tx/txSender'
+import useSafeInfo from '@/hooks/useSafeInfo'
+import EthHashInfo from '@/components/common/EthHashInfo'
+import SendFromBlock from '../../SendFromBlock'
+import { ReviewTokenTxProps, TokenTransferReview } from '@/components/tx/modals/TokenTransferModal/ReviewTokenTx'
+
+const ReviewMultisigTx = ({ params, onSubmit }: ReviewTokenTxProps): ReactElement => {
+  const { safe } = useSafeInfo()
+  const { balances } = useBalances()
+
+  const token = balances.items.find((item) => item.tokenInfo.address === params.tokenAddress)
```

Comment:
I know it was like this before. But could we add a comment somewhere which explains how we handle native transfers (no token address)? Because here my first intuition would be that we could check for a undefined tokenaddress first for native transfers before iterating over the balances.

Reply:
Not sure I understand it correctly but I think the address can't be undefined in this case. Where would you want to add a comment regarding this?

Reply:
Hmm but if we have a native transfer (ETH) there is no token address. What do we use instead here?